bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fullcalendar",
   "title": "FullCalendar",
-  "version": "2.3.2-bcherny",
+  "version": "2.3.2.1-bcherny",
```

Comment:
OK, I'll use `2.3.3`.  My original rationale is that I didn't want to get version numbers mixed up between this fork and upstream.